neuraxle/steps/loop.py

```diff
@@ -285,6 +262,14 @@ def __iter__(self):
         """
         return iter(self.steps_as_tuple)
 
+    def __len__(self):
```

Comment:
Move this method to the new mixin `_HasChildren` with adapted docstring.

Reply:
no, and the place where you put your comment is the reason why we can't do that. the len of a stop cloner != len(children) (yup it is)

Reply:
won't fix.